### ISSUE TITLE
JOSS Paper

### ISSUE THREAD

**Issue by mrava87** [PyLops/pyproximal#155]
Hello everyone,
given that you have contributed some code to PyProximal, I would like to give you the opportunity to be included as co-author of this [JOSS](http://joss.theoj.org) paper. I am writing it mostly to ensure PyProximal can be properly cited by researchers leveraging it in their published work going forward.

I kindly ask you to thumb up if you would like to be included in the author list. If I do not see a thumb up I will assume that you are not interested. 

Moreover, to make the paper stronger, I would like to add a couple of more use-cases of PyProximal: ideally they should be backed up by peer-reviewed publications (and/or openly available codes). It would be great if you could provide me with a use-case from your team or other people you know in your field.

To access the paper please head over to https://github.com/mrava87/pyproximal/actions/runs/7271942518 and click on the `paper` text under Artifacts (the PDF will start downloading). The paper .md file is currently sitting in this branch: https://github.com/mrava87/pyproximal/tree/joss. I am happy to push it to the main repo in a branch with the same name if you want to act directly on it, or just take any of your comments/additions and add it to the paper myself.

Contributors:
@marcusvaltonen
@NickLuiken
@olivierleblanc
@eurunuela
@shnaqvi

**Comment by olivierleblanc** [comment]
Hi Matteo, 

I would greatly appreciate being cited as a co-author, thank you for proposing.
I don't have any comments about the paper.

I have a short use-case of PyProximal in a repository related to recent research in Multi-Core Fiber Lensless Imaging (MCFLI):
[https://github.com/olivierleblanc/interferometric_lensless_imaging/blob/master/reconstruction_demos/1step_strategy/comparisons_model_sparsity_algo/cmp_rec_algo.ipynb](see the code here).

In this Python Notebook, I'm using Pyproximal to solve an inverse problem where the linear operator is the nesting of the Fourier operator, a (kind of) subselection matrix with repetition, then a rank-one projection (ROP) operator. More information about all of this can be found in our paper: [https://arxiv.org/pdf/2306.12698.pdf](https://arxiv.org/pdf/2306.12698.pdf).

Let me know if this would fit.

**Comment by mrava87** [comment]
@olivierleblanc fantastic! This is a great use case, I will add it with link to the paper and repo and ask you to revise the text to make sure I am explaining well the problem and use of proximal operators :D

**Comment by mrava87** [comment]
@olivierleblanc, what about something like this:

```
*Multi-Core Fiber Lensless Imaging*: lensless imaging is a computational imaging technique to reconstruct biological 
  samples at cellular scale. Compressive sensing coupled with rank-one projections has been shown to improve the
  efficiency of the acquisition process [@Leblanc:2023]; this entails solving a $L_1$ regularized inverse problem with
  the proximal gradient method.
```

I am not sure about mentioning the actual notebook, or being a bit more general simply linking the problem that is effective solved to one of PyProximal solvers previously described in the paper. What do you think?

**Comment by marcusvaltonen** [comment]
Hi Matteo,

I would also appreciate being cited as a co-author. Unfortunately, I do not have a peer-reviewed article or code using pyproximal I can share at the moment.

**Comment by olivierleblanc** [comment]
I would rephrase it as:
```
  *Multi-Core Fiber Lensless Imaging* (MCFLI) is a computational imaging technique to reconstruct biological 
  samples at cellular scale. Leveraging the rank-one projected interferometric sensing of the MCFLI has been shown to improve the efficiency of the acquisition process [@Leblanc:2023]; this entails solving a regularized inverse problem with
  the proximal gradient method. Depending on the image to be reconstructed, the regularization term may for instance be $L_1$ or TV. 
```
> I am not sure about mentioning the actual notebook, or being a bit more general simply linking the problem that is effective solved to one of PyProximal solvers previously described in the paper. What do you think?

I agree that mentioning the notebook might not be the best solution as it is part of a bigger project. Linking the problem to the related solver of PyProximal would fit.

**Comment by eurunuela** [comment]
Thank you Matteo. When do you need the feedback for?

**Comment by mrava87** [comment]
I did include you as you had our thumbs up. The paper is now in review https://github.com/openjournals/joss-reviews/issues/6185#issuecomment-1879874325

I can always update the authors list if needed :)

**Comment by eurunuela** [comment]
Thank you! I'm sorry I intended to give feedback but it's been a crazy month.

**Comment by eurunuela** [comment]
I just realized my surname is incorrectly spelled 😅 could you please update it at some point to add a tilde to the n? Thanks 🙏

**Comment by marcusvaltonen** [comment]
I wasn't too bothered but if you change the n -> ñ, perhaps you can change my surname as well from Ornhag -> Örnhag, and Ericsson -> Ericsson Research for affiliation. Thanks!

**Comment by mrava87** [comment]
Alright guys, I can do both at the same time (we are still waiting for an editor to be assigned anyways :) )

**Comment by mrava87** [comment]
Please check that this is now correct :) https://github.com/PyLops/pyproximal/actions/runs/7583340690

**Comment by marcusvaltonen** [comment]
Looks good, thanks!

**Comment by marcusvaltonen** [comment]
@mrava87 Great work getting the paper accepted! Congratulations!

**Comment by mrava87** [comment]
@marcusvaltonen thanks! And thanks everyone for your contribution :D

**Comment by mrava87** [comment]
..and I can happily close this issue 🚀

**Comment by eurunuela** [comment]
Congratulations @mrava87!!!
